lib/LocalVerifiableCredentialStore.js

```diff
@@ -142,4 +170,58 @@ export class LocalVerifiableCredentialStore {
     }
     return typeof issuer === 'string' ? issuer : issuer.id;
   }
+
+  async _queryByExample({credentialQuery}) {
+    if(!credentialQuery) {
+      throw new Error(
+        '"credentialQuery" is needed to execute a QueryByExample.');
+    }
+    if(typeof credentialQuery !== 'object') {
+      throw new Error('"credentialQuery" must be an object or an array.');
+    }
+
+    // normalize query to be an array
+    let query;
+    if(Array.isArray(credentialQuery)) {
+      query = credentialQuery;
+    } else {
+      query = [credentialQuery];
+    }
```

Comment:
You used a nice ternary expression for this on L205, you could do the same thing here.